linux_docker_resources/rti_web_binaries_install_script.py

```diff
@@ -64,7 +65,7 @@ def install_plugin(pkg_path, install_directory):
     try:
         child.expect_exact('Do you want to continue? [Y/n]:')
         child.sendline('y')
-        child.expect(pexpect.EOF)
+        child.expect(pexpect.EOF, timeout=120)
```

Comment:
This seems to increase the timeout from the default of 30 up to 120.  Do we need that?

Reply:
Yeah we definitely do. Installing several of the rtipkgs timed out during test runs.

```
00:00:07.300 Installing Connext...
00:00:07.300 Installing Connext binaries off RTI website...
00:01:02.550 Traceback (most recent call last):
00:01:02.550   File "/tmp/rti_web_binaries_install_script.py", line 67, in install_plugin
00:01:02.551     child.expect(pexpect.EOF)
00:01:02.551   File "/usr/lib/python3/dist-packages/pexpect/spawnbase.py", line 343, in expect
00:01:02.557     return self.expect_list(compiled_pattern_list,
00:01:02.557   File "/usr/lib/python3/dist-packages/pexpect/spawnbase.py", line 372, in expect_list
00:01:02.557     return exp.expect_loop(timeout)
00:01:02.557   File "/usr/lib/python3/dist-packages/pexpect/expect.py", line 181, in expect_loop
00:01:02.565     return self.timeout(e)
00:01:02.565   File "/usr/lib/python3/dist-packages/pexpect/expect.py", line 144, in timeout
00:01:02.565     raise exc
00:01:02.565 pexpect.exceptions.TIMEOUT: Timeout exceeded.
00:01:02.565 <pexpect.pty_spawn.spawn object at 0x7fba90548a90>
00:01:02.565 command: /home/rosbuild/rti_connext_dds-6.0.1/bin/rtipkginstall
00:01:02.565 args: [b'/home/rosbuild/rti_connext_dds-6.0.1/bin/rtipkginstall', b'/tmp/rticonnextdds-src/rti_connext_dds-6.0.1.25-pro-host-x64Linux.rtipkg']
00:01:02.565 buffer (last 100 chars): 'g rti_connext_dds-6.0.1.25-pro-host-x64Linux.rtipkg. Please wait, this may take several minutes...\r\n'
```